content/docs/clients/javascript/index.md

```diff
@@ -36,12 +29,33 @@ To install via a script tag referencing Exceptionless over a CDN, add the follow
 ```html
 <script type="module">
   import { Exceptionless } from "https://unpkg.com/@exceptionless/browser";
-
-  await Exceptionless.startup(c => {
-    c.apiKey: 'API_KEY_HERE'
-  });
 </script>
-```html
+```
+
+Next, you just need to call startup during your apps startup to automatically capture unhandled errors.
+
+```js
+import { Exceptionless } from "@exceptionless/browser";
```

Comment:
This really depends on what one you import, if you are importing from a cdn, line 38 won't work.

Reply:
Think the next line needs to be more specific or what? The line right below line 38 is: 

```
// Or import { Exceptionless } from "https://unpkg.com/@exceptionless/browser";
```

Reply:
This section is specifically for using from a CDN so I don't think we should be showing code that uses an installed NPM package.

Reply:
I think the problem is that we start getting into a more complex sample with setting up configuration and default data and tags. We were only doing that in the CDN section previously. Probably should break the additional config out into it's own section or not even talk about that stuff in this page.

Reply:
You're right, @ejsmith, I don't think config values need to even show up here. It's a quick start guide, so the only thing needed is the API key. We have the next two sections dedicated to configuration and values. 

Let me know if that works better for you. Looks a lot clearer to me.